client/router_create.go

```diff
@@ -1237,6 +1242,14 @@ sasldb_path: /tmp/skrouterd.sasldb
 		}
 	}
 
+	var disableEvents = options.Spec.DisableSkupperEvents
+	err = kube.AddEventRecorderPermissions(van.Namespace, ownerRefs, cli.KubeClient, types.ControllerServiceAccountName)
+	if err != nil {
+		disableEvents = true
```

Comment:
I feel we should at least log the error here. Alternatively we could return the error and therefore require events to be explicitly disabled. Or we could flip the default for events, and require them to be explicitly enabled, and again return the error if this was not possible.

Reply:
I like the idea of kubernetes events being generated by default. And considering that, I agree that
if events cannot be generated, users must be aware of that. A warning message on stdout or at least
an error message logged during the service controller startup.

Reply:
Updated in 0a62b447a69ae400b1b0326680a65a6e2f13bbde